### ISSUE TITLE
Remove excess documentation for BasicServer GET (Authenticated)

### ISSUE THREAD

**Issue by jleung51** [jleung51-coursework/phaser#25]
BasicServer's `handle_get()` has documentation for authenticating operations which should not have an authentication option.

For example, when retrieving the contents of a table,

```
http://localhost:34568/ReadEntityAuth/TABLE_NAME/AUTHENTICATION_TOKEN
```

should **not** be possible.

**Comment by jleung51** [comment]
Working on it in the branch `basic-get-auth`!

**Comment by daphne-chong** [comment]
Fixed in pull request #26
